microkernel.ks

```diff
@@ -166,6 +166,13 @@ EOMEI
 echo " * compressing cracklib dictionary"
 gzip -9 /usr/share/cracklib/pw_dict.pwd
 
+echo " * setting up journald and tty1"
+rm -f /etc/systemd/system/getty.target.wants/getty@tty1.service
```

Comment:
I'd remove this; we want the user to be able to log into the system, to run the `mk` command, troubleshoot driver issues, etc. if needed.